core/blockchain_snapshot_test.go

```diff
@@ -150,6 +150,7 @@ func (basic *snapshotTestBasic) verify(t *testing.T, chain *BlockChain, blocks [
 	}
 }
 
+//nolint:unused
```

Comment:
And here too @karalabe, lot of code that is unused, seems like you coded up a test but forgot to enable it?